matter/mbedtls/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
```

Comment:
This file already exists in the matter repo and is used to build a mbedtls exists. If anything you should leverage the mbedtls_target usage.